pennylane/ops/op_math/composite.py

```diff
@@ -65,6 +65,17 @@ def __init__(
         if do_queue:
             self.queue()
 
+        self._check_batch_sizes()
```

Comment:
I would prefer to call this `_set_batch_size` since that's ultimately what it's hoping to do. Also, is it in its own method just for cleanliness? not picky either way, just curious

Reply:
Changed!